src/dlgProfilePreferences.cpp

```diff
@@ -21,11 +21,17 @@
  *   59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.             *
  ***************************************************************************/
 
+/*
+ * THIS FILE CONTAINS UTF-8 UNICODE ENCODED CHARACTER STRINGS THAT ARE (OR
```

Comment:
I'm not sure what is the takeaway message I'm supposed to get here, could you explain?

Reply:
The tool-tips in the constructor MUST remain as `QStringLiterals(`...`)` and NOT be converted to `tr(`...`)` but I could word it as that now - when I started on this I was not quite sure how it would all work out... 🙂

Reply:
Okay - can we just say that, without the shouting?

Reply:
~~YES~~ err,  yes!

Reply:
☑️ Done